internal/config/config.go

```diff
@@ -38,6 +38,15 @@ func MustEnableUnitTestingMode() {
 }
 
 func EnableUnitTestingMode() error {
+	return EnableUnitTestingModeFull(true)
+}
+
+// - Sets configuration options
+// - Changes base directory to a temp directory
+// - Sets logging to debug if not already set
+// - Copies test data into the working directory.
```

Comment:
I don't like how the first sentence (which is supposed to be a summary of the function) is a list.

Also, you list is inconsistent (one ends with a period).